test/common/upstream/zone_aware_load_balancer_fuzz_base.h

```diff
@@ -0,0 +1,50 @@
+#include "test/mocks/upstream/priority_set.h"
+
+#include "load_balancer_fuzz_base.h"
+
+namespace Envoy {
+namespace Upstream {
+class ZoneAwareLoadBalancerFuzzBase : public LoadBalancerFuzzBase {
+public:
+  ZoneAwareLoadBalancerFuzzBase(bool need_local_cluster, const std::string& random_bytestring)
+      : LoadBalancerFuzzBase(), random_bytestring_(random_bytestring) {
+    if (need_local_cluster) {
+      local_priority_set_ = std::make_shared<NiceMock<MockPrioritySet>>();
+      local_priority_set_->getMockHostSet(0);
+    }
+    // If you call local_priority_set_.get() and it's not constructed yet, it'll return a nullptr
```

Comment:
is this comment still needed?

Reply:
Deleted. This was actually supposed to map to people reading the check I had, but you're right especially with using the boolean operator of the shared_ptr, it's implied.